### ISSUE TITLE
[Visual Requirements - Make Code - Import URL] Luminosity ratio is less than 4.5:1 for text control present on 'Open project URL' pop-up.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3552]
**User Experience:**
Visual impaired user would find difficulty to see the control or element if that controls have insufficient color contrast as they won't be able to take on that controls and he/she would miss the functionality of such control or element.

**Note**: User credentials should NOT be included in the bug.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate to "Import" button and select it.
4. Navigate through the controls appearing under "Import" dialog.
5. Navigate to Import URL option and navigate through the all the controls of dialog.
6. Verify contrast ratio of text control(The content below is provided by a user, and is not endorsed by Microsoft.
If you think it's not appropriate, please report abuse through Settings -> Report Abuse.)

**Actual Result:**
Luminosity ratio is less than 4.5:1 for text control(The content below is provided by a user, and is not endorsed by Microsoft.
If you think it's not appropriate, please report abuse through Settings -> Report Abuse.) under 'Open project URL' pop-up.

**Expected Result:**
Luminosity ratio should be more than and equal to 4.5:1 for text control(The content below is provided by a user, and is not endorsed by Microsoft. If you think it's not appropriate, please report abuse through Settings -> Report Abuse.) under 'Open project URL' pop-up .

**Note:**
Same issue exists on Firefox & Chrome.

**WCAG Reference:**
[WCAG 1.4.3 - Non-text Contrast
](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html)

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Tool: CCA

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev2, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG1.4.3, #Win10-Edge(Chromium)

![WCAG 1 4 3_Import URL_CCA](https://user-images.githubusercontent.com/72871089/97857710-2c0cda00-1d24-11eb-93c8-513482325ab5.jpg)
